### PR TITLE
DEV-3188-new-go-exercise-halfslice

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

**Review by jotapero — CHANGES_REQUESTED** [review]

File: subjects/halfslice/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## half-Slice
```
[the remark this answers is not in the document]
Comment (Hamzaelkhatri) [reply subjects/halfslice/README.md]: fixed , thank you

File: subjects/halfslice/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+```go
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println(HalfSlice([]int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10}))
```
[the remark this answers is not in the document]
Comment (Hamzaelkhatri) [reply subjects/halfslice/README.md:27]: fixed

**Review by jotapero — APPROVED** [review]
All good

**Review by jotapero — APPROVED** [review]

**Review by jotapero — APPROVED** [review]

**Review by jotapero — APPROVED** [review]
Looks Good :alien:

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**